core/src/App.html

```diff
@@ -545,9 +547,37 @@
         });
       },
       onResize() {
-        if (window.innerWidth >= CSS_BREAKPOINTS.desktopMinWidth) {
+        const simpleDesktopToMobile = () => {
+          if (this.get().responsiveNavSetting == 'simple') {
```

Comment:
Please always use `===` instead of `==`. You will be always be right using `===`